### ISSUE TITLE
Clavicular pseudarthrosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7349]
**Preferred term label:**
Clavicular pseudarthrosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
A developmental defect in a the clavicule leading to bending and pathologic fracture, with inability to form a normal bony callus with subsequent fibrous nonunion, leading to the pseudarthrosis (or "false joint").

**Parent term (use hpo.jax.org/app)**
Pseudoarthrosis HP:0005864

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIMN:156530 
PMID: 19232556   1/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
